File: C/day046.c
/* Author: Ahmed A. M. ALOMAR
Date: February 15 - 2020
Description:
This program is an example of the break statement in a loop  */


#include <stdio.h>

int main()
{
  int x = 0;
  while (x < 10)
  {
    if (x == 5)
    {
      break; // THe loop will break here
    }
    printf("x = %d \n", x);
    x++;
  }
  return 0;
}

File: SourceCode/Python/day046.py
# ------------------------------------ #
# Author: Ahmed A. M. ALOMAR
# Date: May 26 - 2020
#
# Description:
# * Example of Class Methods
# ------------------------------------ #


class GemStones():
  def __init__(self, name, color):
    self.name = name
    self.color = color

  def nice(self):
    print("A nice Gem Stone")

  def details(self):
    print(f"This {self.name} is colored {self.color}.")


gem_one = GemStones("Ruby", "Red")
gem_two = GemStones("Jade", "Green")
gem_three = GemStones("Sapphire", "Blue")

print(gem_one.name)
print(gem_two.color)

gem_three.details()
gem_three.nice()
